swdb_2018_tools/GLM_helpers.py

```diff
@@ -0,0 +1,98 @@
+import statsmodels.api as sm
+import numpy as np
+import pandas as pd
+import keras
+from keras import backend as K
+from keras.models import Model
+from keras.layers import Input
+from keras import Sequential
+from keras.layers import Dense, Lambda
+
+# Inputs:
```

Comment:
Good on you for documenting your functions. You might find this easier if you use block comments [like here](https://github.com/numpy/numpy/blob/master/numpy/core/numeric.py#L208).